webplugin/js/app/kommunicate.js

```diff
@@ -931,4 +931,26 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getCurrentPosition:() => 
+        new Promise((resolve, reject) => 
+          navigator.geolocation.getCurrentPosition(resolve, reject)
+        ),
+    getUserLocation: async function (){
+      try {
+        
+          const api_key='AIzaSyAGVIsWxU7lkCuoodgI6FGXmDN5J11VJFk';
+          const position=await this.getCurrentPosition();
+          MCK_CURR_LATITIUDE=position.coords.latitude;
+          MCK_CURR_LONGITUDE=position.coords.longitude;
+          const response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?latlng=${MCK_CURR_LATITIUDE},${MCK_CURR_LONGITUDE}&sensor=true&key=${api_key}`);
+          const result = await response.json();
+          if (result && result.results.length) {
+                return result.results[0].formatted_address;
+          }
+          return "location not found";
```

Comment:
where will this "location not found" be used? will it be shown to end user?

Reply:
if we are unable to find the location then it will show "location not found" instead of actual location on right side of the dashboard to the end user.

Reply:
keep it blank, in that case, we will not pass the location

Reply:
return "LOCATION_NOT_FOUND"